composer.json

```diff
@@ -31,7 +31,7 @@
     "oat-sa/generis" : ">=14.0.0",
     "oat-sa/tao-core" : ">=47.0.0",
     "oat-sa/extension-tao-lti" : ">=15.7.1",
-    "oat-sa/extension-tao-delivery" : ">=15.0.0",
+    "oat-sa/extension-tao-delivery" : ">=15.13.0",
```

Comment:
it's not released extension version yet?

Reply:
yes

Reply:
is it a good approach to use not existed version numbers? some body can bump version of target extension while you working on it and take this version for his changes. and when you start release your changes the version can be different. You can miss this situation and here accidentally can be situation with miss-matching of minimal required version

Reply:
These 3 PR should come together @wazelin recommended change it now [here](https://github.com/oat-sa/extension-tao-deliver-connect/pull/132#pullrequestreview-1111820119)

Reply:
releasing them together doesn't exclude described in my comment situation.

> An updated dependency on taoDelivery extension

i think it's mean that for a moment you can use dev dependency branch just do not miss this situation and use correct version just after releasing.

i'm not requiring any changes now, in case if we will pay attention for such situation. But IMO it's a not good approach pointing on not released version.

Reply:
Lets's then not merge it till the version bumped in PR https://github.com/oat-sa/extension-tao-delivery/pull/555 we will see it when unit test for it passed